src/changes.go

```diff
@@ -464,6 +465,10 @@ func (g *Commands) changeSlice(cslArg *changeSliceArg) {
 		localBase := remotePathJoin(cslArg.localParent, l.Name())
 		remoteBase := remotePathJoin(cslArg.remoteParent, l.Name())
 
+		if l.remote != nil && (g.opts.Decrypter != nil || g.opts.Encrypter != nil) && !l.remote.IsDir {
```

Comment:
Nice! Good catch.

We could break this out first in `src/commands.go`
```go
func (opts *Options) CryptoEnabled() bool {
   return opts.Decrypter != nil || opts.Encrypter != nil
}
```

and then  here

```go
nonDirRemote := l.remote != nil && !l.remote.IsDir 
if nonDirRemote && g.opts.CryptoEnabled() {
   l.remote.Size -= int64(dcrypto.Overhead)
}
```